materialize-cratedb/CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# materialize-postgres
```

Comment:
This changelog file is not relevant to the new connector - it can just be deleted for now for cratedb.